src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -1,5 +1,10 @@
 package seedu.address.logic.commands;
 //@@author TanYikai
```

Comment:
haha this annotation should be right above the javadoc

Reply:
I dont understand what you meant, could you elaborate thanks